waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3884 @@
+"Mosch-Kogel",Moschgel,AT,4649.333N,01459.583E,1916.0m,7,,,,"Gipfel"
+"Moserhoerndl",Moserndl,AT,4724.966N,01332.833E,1494.0m,7,,,,"Gipfel"
+"Mosermandl",Moserndl,AT,4712.366N,01324.483E,2680.0m,7,,,,"Gipfel"
+"Mosgan",Mosgan,AT,4629.466N,01442.383E,1575.0m,7,,,,"Gipfel"
+"Muehlhauserhoehe",Muehlhoehe,AT,4701.411N,01346.811E,2214.0m,7,,,,"Gipfel"
+"Mulle",Mulle,AT,4657.083N,01209.283E,3159.0m,7,,,,"Gipfel"
+"Mulleter-Seichen-Kogel",Mullegel,AT,4654.933N,01249.966E,2916.0m,7,,,,"Gipfel"
+"Muntanitz",Muntaitz,AT,4704.449N,01235.116E,3232.0m,7,,,,"Gipfel"
+"Murkar-Spitze",Murkatze,AT,4701.466N,01102.016E,3148.0m,7,,,,"Gipfel"
+"Mutmal-Spitze",Mutmatze,AT,4647.133N,01054.700E,3528.0m,7,,,,"Gipfel"
+"Muttekopf",Mutteopf,AT,4716.166N,01039.100E,2777.0m,7,,,,"Gipfel"
+"Muttjoechle",Muttjhle,AT,4705.941N,01001.591E,2073.9m,7,,,,"Gipfel"
+"Muttler-Kogl",Muttlogl,AT,4653.883N,01036.350E,2747.0m,7,,,,"Gipfel"
+"Naaf-Kopf",Naaf-opf,AT,4703.683N,00936.450E,2571.0m,7,,,,"Gipfel"
+"Napf-Joch",Napf-och,AT,4703.616N,01202.316E,3143.0m,7,,,,"Gipfel"
+"Nassfeld / Pass",Nassfass,AT,4633.616N,01316.583E,1530.0m,6,,,,"Pass"
+"Nauderer Hennesiegel-Sp.",NaudeSp.,AT,4651.950N,01037.283E,3045.0m,7,,,,"Gipfel"
+"Nazogl",Nazogl,AT,4736.700N,01413.666E,2055.0m,7,,,,"Gipfel"
+"Nebelkareck",Nebeleck,AT,4708.599N,01320.100E,2535.0m,7,,,,"Gipfel"
+"Neuschmiel",Neusciel,AT,4714.649N,01238.600E,2121.0m,7,,,,"Gipfel"
+"Neves Sattel",Nevestel,AT,4658.533N,01145.833E,3027.0m,7,,,,"Gipfel"
+"Noerderkogl",Noerdeogl,AT,4654.313N,01100.516E,3166.0m,7,,,,"Gipfel"
+"Noerdl. Granatenkogl",Noerdlogl,AT,4651.516N,01104.233E,3056.0m,7,,,,"Gipfel"
+"Noerdl. Woester-Spitze",Noerdltze,AT,4712.833N,01011.566E,2558.0m,7,,,,"Gipfel"
+"Noesslach-Joch",Noesslaoch,AT,4703.466N,01126.433E,2231.0m,7,,,,"Gipfel"
+"Noppen-Spitze",Noppetze,AT,4719.733N,01026.633E,2596.0m,7,,,,"Gipfel"
+"Nussing-Kogel",Nussigel,AT,4703.066N,01233.166E,2991.0m,7,,,,"Gipfel"
+"Ob. Wartkopf",Ob.Waopf,AT,4710.650N,01216.950E,2640.0m,7,,,,"Gipfel"
+"Ob. Zar-Spit",Ob.Zapit,AT,4650.733N,01233.750E,2778.0m,7,,,,"Gipfel"
+"Oberer Eisenkopf",Obereopf,AT,4713.133N,01038.666E,2337.0m,7,,,,"Gipfel"
+"Oblitzen",Oblitzen,AT,4704.849N,01327.466E,2657.0m,7,,,,"Gipfel"
+"Ochsenberg",Ochseerg,AT,4744.539N,01312.755E,1483.0m,7,,,,"Gipfel"
+"Ochsen-Kogel",Ochsegel,AT,4716.500N,01204.650E,2470.0m,7,,,,"Gipfel"
+"Ochsen-Kopf",Ochseopf,AT,4656.800N,01251.500E,2610.0m,7,,,,"Gipfel"
+"Ochsen-Kopf VA",OchsefVA,AT,4706.839N,00937.487E,2286.0m,7,,,,"Gipfel"
+"Ochsenriegel",Ochsegel,AT,4659.101N,01347.890E,2282.0m,7,,,,"Gipfel"
+"Ochsner",Ochsner,AT,4702.683N,01148.883E,3107.0m,7,,,,"Gipfel"
+"Oedkar-Spitze",Oedkartze,AT,4724.766N,01125.583E,2743.0m,7,,,,"Gipfel"
+"Oestl. Feuerstein",Oestl.ein,AT,4658.499N,01114.266E,3268.0m,7,,,,"Gipfel"
+"Oestl. Karwendel-Spitze",Oestl.tze,AT,4726.716N,01125.300E,2538.0m,7,,,,"Gipfel"
+"Oestl. Knoten-Spitze",Oestl.tze,AT,4703.866N,01110.916E,3100.0m,7,,,,"Gipfel"
```

Comment:
abbreviations

Reply:
I kept the abbreviations for noerdl. östl. westl. südl.